### DISCUSSION TITLE
yarn generate error exist code 1

### DISCUSSION THREAD

**Discussion by SlimGuenidez** [prisma/prisma#19652] [Q&A]
### Bug description

Hello , when i try to run the command generate i get this error : 

> Downloading Prisma engines for Node-API for windows [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/8fbc245156db7124f997f4cecdd8d1219e360944/windows/query_engine.dll.node.gz.sha256 failed, reason: connect ETIMEDOUT 172.67.4.173:443
error Command failed with exit code 1.

i tried  to clean the cache and removing the nodes modules.... 

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by petradonka** [comment]
Hey @SlimGuenidez: are you able to access this URL in your browser?

**Comment by performautodev** [comment]
Same error +1:  


$ npx prisma migrate dev
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": PostgreSQL database "postgres", schema "public" at "localhost:5752"

Already in sync, no schema change or pending migration was found.

Running generate... (Use --skip-generate to skip the generators)
Error: request to https://binaries.prisma.sh/all_commits/548464...44456/windows/query_engine.dll.node.gz.sha256 failed, reason: connect ETIMEDOUT 104.22.23.109:443

**Reply by performautodev** [reply]
Downloadning the file work fine but using CLI not working after search i think the IP belong to cloudflare:

![image](https://github.com/prisma/prisma/assets/62381545/3137ed4b-9627-4d15-9921-5e0ad5ec9897)

![image](https://github.com/prisma/prisma/assets/62381545/9ff83dab-8fa1-4063-a3ae-da0f166fa2e2)

**Reply by petradonka** [reply]
Thanks for letting us know that the same URL works fine in the browser!

Do you have any special networking characteristics that might interfere with accessing Cloudflare?

**Reply by performautodev** [reply]
yeh i think is something related to cloudflare not to prisma

**Reply by tihomirovkirillcom** [reply]
Formula/game-porting-toolkit-compiler.rb

**Reply by nurul3101** [reply]
I'll mark this discussion as closed as it looks like a network related issue and not related to Prisma.

**Comment by performautodev** [comment]
same :

$ npx prisma generate
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Error: request to https://binaries.prisma.sh/all_commits/8fbc....360944/windows/query_engine.dll.node.gz.sha256 failed, reason: connect ETIMEDOUT 104.22.23.109:443
